forms-flow-web/src/components/Modeler/DecisionTable.js

```diff
@@ -0,0 +1,287 @@
+            <thead className="table-header">
+              <tr>
+                <th className="w-25" scope="col" >
+                 <SortableHeader
+                   columnKey="name"
+                   title="Name"
+                   currentSort={currentDmnSort}
+                   handleSort={handleSort}
+                   className="ms-4"
+                  />
+                  </th>
+                  <th className="w-20" scope="col">
+                  <SortableHeader
+                   columnKey="id"
+                   title="ID"
+                   currentSort={currentDmnSort}
+                   handleSort={handleSort}
+                  />
+                  </th>
+                  <th className="w-15" scope="col">
+                  <SortableHeader
+                   columnKey="modified"
+                   title="Last Edited"
+                   currentSort={currentDmnSort}
+                   handleSort={handleSort}
+
+                  />
+                  </th>
+                  <th className="w-15" scope="col">
+                  <SortableHeader
+                   columnKey="status"
+                   title="Status"
+                   currentSort={currentDmnSort}
+                   handleSort={handleSort}
+                  />
+                  </th>
+                <th
+                  className="w-25"
+                  colSpan="4"
+                  aria-label="edit bpmn button "
+                ></th>
+              </tr>
+            </thead>
+            <tbody>
+            {dmn.map((dmnItem) => (
+              <tr key={dmnItem.id}>
+                <td className="w-25">
+                  <span className="ms-4">{dmnItem.name}</span>
+                </td>
+                <td className="w-20">
+                  <span className="">{dmnItem.parentProcessKey}</span>
+                </td>
+                <td className="w-15">
+                  {HelperServices?.getLocaldate(dmnItem.modified)}
+                </td>
+                <td className="w-15">
+                  <span
+                    data-testid={`sub-flow-status-${dmnItem._id}`}
+                    className="d-flex align-items-center"
+                  >
+                    {dmnItem.status === "active" ? (
+                      <>
+                        <span className="status-live"></span>
+                      </>
+                    ) : (
+                      <span className="status-draft"></span>
+                    )}
+                    {dmnItem.status === "active"
+                      ? t("Live")
+                      : t("Draft")}
+                  </span>
+                </td>
+                <td className="w-25">
+                  <span className="d-flex justify-content-end">
+                    <CustomButton
+                      variant="secondary"
+                      size="sm"
+                      label={<Translation>{(t) => t("Edit")}</Translation>}
```

Comment:
Cann you replace <Translation>{(t) => t("Edit")}</Translation>}

with {t("Edit")} 

Similarly in the entire page . @fahad-aot

Reply:
Updated.